build_mpq.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+SCRIPTDIR="$(readlink -f $0)"
+SCRIPTDIR="$(dirname "${SCRIPTDIR}")"
+
+PSXDIR=$SCRIPTDIR/ps1_assets
+
+DSTREAM=$SCRIPTDIR/dstream.bin
+DBANK=$SCRIPTDIR/dbank.bin
+VAG2WAV=$SCRIPTDIR/vag2wav.bin
+
+
+TEMPDIR=$(mktemp -d)
+#trap "rm -rf $TEMPDIR" EXIT
+
+cd $TEMPDIR
+
+for DIR in $PSXDIR/*.dir; do
+    TMP=$(basename $DIR)
+    mkdir -p $TMP
+
+    (cd $TMP; $DSTREAM $DIR ${DIR::-4}.bin > /dev/null)
+
+    for VAG in $TMP/*.VAG; do
+	$VAG2WAV $VAG ${VAG::-4}.WAV > /dev/null
```

Comment:
Looks like there is a mix between the use of spaces and tabs for indentation here? Besides that, great PR! :) It will be exciting to add new audio contents to mods.